### ISSUE TITLE
Pytorch `tensor.dcp()` to be a shorthand for `my_tensor.detach().cpu().numpy()`

### ISSUE THREAD

**Issue by dsm-72** [pytorch/pytorch#91382]
### 🚀 The feature, motivation and pitch

I can't tell you how many times I call something to the effect of `my_tensor.detach().cpu().numpy()` somewhere. Convenience function `my_tensor.dcp()` would be nice. 

### Alternatives

`my_tensor.numpy(detach=True, cpu=True)` with these being default to true

cc @mruberry @rgommers

**Comment by vadimkantorov** [comment]
I think `.numpy(force=True)` does this already, no?

**Comment by rgommers** [comment]
@vadimkantorov's answer is correct, so I'll close this issue.
